src/foam/comics/v2/DAOSummaryView.js

```diff
@@ -150,30 +150,31 @@ foam.CLASS({
 
       this
         .addClass(this.myClass())
-        .add(self.slot(function(data, data$id, config$CRUDActionsAuth$update, config$CRUDActionsAuth$delete, config$browseTitle, config$viewBorder, viewView) {
-
-          // iterate through permissions and replace % with data$id
-          var editAction = self.EDIT;
-          var deleteAction = self.DELETE;
-
-          if ( config$CRUDActionsAuth$update ) {
-            var editArray = config$CRUDActionsAuth$update;
-
-            editArray = editArray.map((permission) => permission.replace('%', data$id));
-
-            editAction = self.EDIT.clone().copyFrom({
-              availablePermissions: self.EDIT.availablePermissions.concat(editArray)
-            });
-          }
-
-          if ( config$CRUDActionsAuth$delete ) {
-            var deleteArray = config$CRUDActionsAuth$delete;
-
-            deleteArray = deleteArray.map((permission) => permission.replace('%', data$id));
-
-            deleteAction = self.DELETE.clone().copyFrom({
-              availablePermissions: self.DELETE.availablePermissions.concat(deleteArray)
-            });
+        .add(self.slot(function(data, data$id, config$viewBorder, viewView) {
+
+          if ( ! self.config.CRUDPermission) {
+            self.config.editEnabled = true;
```

Comment:
If we are retaining editEnabled (and others above) for backwards compatibility (I presume), then self.config.editEnabled is either set or already has a default.